potassium/potassium.py

```diff
@@ -162,6 +166,8 @@ def _handle_generic(self, endpoint, flask_request):
                 res = make_response(tb_str)
                 res.status_code = 500
                 res.headers['X-Endpoint-Type'] = endpoint.type
+            self._idle_start_time = time.time()
+            self._inference_start_time = 0
```

Comment:
we're essentially using _inference_start_time=0 as a sentinel value here which i don't think is ideal (setting start time to be 0 ms since epoch doesn't really make semantic sense).

We may want to consider renaming this value to `_last_inference_start_time` so we don't feel the need to update it when releasing the lock

Reply:
Yes, very good point. Latest change makes:
- the var is called `_last_inference_start_time`
- sets it to none if it's not initialised
- still sends a 0 in the json response if `_last_inference_start_time==None`